### DISCUSSION TITLE
How to activate validation for required field for initial value?

### DISCUSSION THREAD

**Discussion by yhy-1** [ant-design/ant-design#41130] [Q&A] [answered]
I have the required set for all the form items. There will be null and undefined values that will come back from the database that will be set in the initial value due to these values having an expiration date so it can change. I can't seem to find any way to get it to work to display the error message that it is a required form item.  The form.validateField only works when interacting with the form item. Unless the only way is to the getfieldValue and check if it is null/undefined and set help in the form item. Which is not really ideal.

**Comment by zombieJ** [comment, marked as answer]
useEffect + validateFields

**Reply by yhy-1** [reply]
Thanks, did more debugging. It looks like we used useWatch for the show/hide components didn't work well with validateFields for the initial value. It works now, thank you.
